nasa-gibs/worldview
# WV-3154 Replace Social Tab with Citation Tab

## Description
This change removes the social tab from the Share button and replaces it with a Citation tab.

## How To Test
1. `git checkout wv-3154-citation-tab`
2. `npm ci`
3. `npm run watch`
4. Open the citation tab via the share icon at the top-right of WV
5. Verify the text citation tab looks like it fits-in, and the text can be copied from it via the Copy button
